site/content/3.13/data-science/graphml/ui.md

```diff
@@ -0,0 +1,254 @@
+of graph nodes that capture structural and feature-based information.
+
+**Use cases include:**
+- Similarity search (e.g. finding similar products, users, or documents)
+- Link prediction (e.g. suggesting new connections)
+
+**Configuration parameters:**
+- **Type of Training Job:** Node embeddings
+- **Target Vertex Collection:** Select the collection to generate embeddings for (e.g. `movie` or `person`)
+- No label is required for training in this mode
+
+**Understanding Vertex Collections:**
+
+- **X Vertex Collection:** These are the source nodes used during training.
+  They represent the full set of nodes on which features were computed (e.g. `person`, `movie`).
+- **Y Vertex Collection:** These are the target nodes that contain labeled data.
+  The labels in this collection are used to supervise the training process and
+  are the basis for evaluating prediction quality.
+
+The target collection is where the model's predictions are stored when running a prediction job.
+
+Once the configuration is complete, click **Begin training** to start the embedding job.
+
+![Node Embeddings](../../../images/ml-node-embedding.png)
+
+## Model selection phase
+
+Once the training is finished, the job status updates to **READY FOR MODEL SELECTION**.
+This means the model has been trained using the provided vertex and edge data
+and is now ready for evaluation.
+
+A list of trained models is displayed, along with performance metrics
+(Accuracy, Precision, Recall, F1 score, Loss). Review the results of different
+model runs and configurations.
+
+![GraphML Model Selection](../../../images/graph-ml-model.png)
+
+Select the best performing model suitable for your prediction task. You can also
+open the **Confusion Matrix** to compare predicted values versus actual values.
+
+![GraphML Confusion Matrix](../../../images/graphml-ui-confusion-matrix.png)
+
+## Prediction phase
+
+After selecting a model, you can create a Prediction Job. The Prediction Job
+generates predictions and persists them to the source graph, either in a new
+collection or within the source documents.
+
+The Prediction interface allows inference to be run using the selected model.
+It enables configuration of how predictions are executed, which collections are
+involved, and whether new or outdated documents should be automatically featurized
+before prediction.
+
+You have two important options:
+
+- **Featurize new documents:** Enable this option to generate features for
+  documents that have been added since the model was trained. This is useful
+  for getting predictions on new data without having to retrain the model.
+- **Featurize outdated documents:** Enable this option to re-generate features
+ for documents that have been modified. Outdated documents are those whose
+ attributes (used during featurization) have changed since the last feature
+ computation. This ensures your predictions reflect the latest changes to your data.
```

Comment:
```suggestion
- **Featurize outdated documents:** Enable this option to re-generate features
  for documents that have been modified. Outdated documents are those whose
  attributes (used during featurization) have changed since the last feature
  computation. This ensures your predictions reflect the latest changes to your data.
```